src/lib.rs

```diff
@@ -446,6 +446,22 @@ make_struct!(
     struct Level5PagingRequest: [0x94469551da9b3192, 0xebe5e86db7382888] => Level5PagingResponse {};
 );
 
+// Paging mode request tag:
+#[repr(C)]
+#[derive(Debug)]
+pub struct PagingModeResponse {
+    pub revision: u64,
+    pub mode: u64,
```

Comment:
It would be nice to have the paging mode mode's be added as an enum. Use `#[cfg(target_arch = "xxx")]` to determine which flags the enum would have and then add `#[repr(u64)]` on top of the enum to ensure the sizeof it.

Reply:
What about the default values, impl Default for a PagingMode enum does not work because it would need to be const?

Reply:
Yes :)